packages/frontend/src/connectors.ts

```diff
@@ -1,15 +1,14 @@
 import { configureChains } from 'wagmi';
-import { mainnet, polygon } from 'wagmi/chains';
 import { publicProvider } from 'wagmi/providers/public';
 import { InjectedConnector } from 'wagmi/connectors/injected';
 import { WalletConnectConnector } from 'wagmi/connectors/walletConnect';
 import { CoinbaseWalletConnector } from 'wagmi/connectors/coinbaseWallet';
 import { walletConnectProjectId } from './config';
-
+import { SUPPORTED_CHAINS } from './utils/chains';
 
 type SupportedWallet = 'injected' | 'coinbase' | 'walletconnect';
 
-const { chains } = configureChains([mainnet, polygon], [publicProvider()]);
+const { chains } = configureChains([...SUPPORTED_CHAINS], [publicProvider()]);
```

Comment:
Weird, I guess this doesn't actually matter if it hasn't been changed until now?

Reply:
It didn't matter for some chains, but did matter for Base for some reason.